context.go

```diff
@@ -669,18 +776,18 @@ loop:
 			if tooManyErrors {
 				continue
 			}
-			// Limit concurrent calls to parseBlueprintFiles to 200
-			// Darwin has a default limit of 256 open files
-			if activeCount >= 200 {
-				pending = append(pending, blueprint)
-				continue
-			}
-			startParseBlueprintsFile(blueprint)
-		case <-doneCh:
+			foundParseableBlueprint(blueprint)
+		case blueprint := <-doneCh:
 			activeCount--
-			if len(pending) > 0 {
-				startParseBlueprintsFile(pending[len(pending)-1])
+			deps = append(deps, blueprint.string)
```

Comment:
Can this go in startParseDescendants instead?

Reply:
done